samples/net/http_server/boards/native_sim.conf

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# LED Control off
+CONFIG_DK_LIBRARY=n
+
+# Network address config
+CONFIG_NET_CONFIG_SETTINGS=y
+CONFIG_NET_CONFIG_NEED_IPV4=y
+CONFIG_NET_CONFIG_MY_IPV4_ADDR="192.0.2.1"
+CONFIG_NET_HOSTNAME_ENABLE=y
+CONFIG_NET_HOSTNAME_UNIQUE=n
+CONFIG_NET_HOSTNAME="httpserver"
+CONFIG_POSIX_UNAME=n
+
+# DNS
+CONFIG_DNS_RESOLVER=y
+CONFIG_DNS_SERVER_IP_ADDRESSES=y
+CONFIG_DNS_SERVER1="8.8.8.8"
```

Comment:
Do we need this? AFAIK we don't do any DNS request from the server code that require this.
I also don't see mDNS nor SD being enabled. Something that we use for in the Wi-Fi config to be able to be addressed with the hostname in HTTP requests.

Reply:
yes you are right. Not needed